app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
1. Lỗi Coding convention:
+ Dùng nháy đôi trong ruby nhé
+ Các thành phần tách nhau 1 space thôi nhé

2. EM tìm hiểu về `I18n in rails ` nhé
Sử dụng I18n cho toàn bộ text ở các file `html` nhé em